.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,22 @@
 # Contributing
+![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
 
-Please see instructions for contributing here:
-https://checkstyle.org/contributing.html
+#### [Reporting issues](https://checkstyle.org/report_issue.html)
+#### [Pull Request Template](https://github.com/checkstyle/checkstyle/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+#### [Issue Template](https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md)
```

Comment:
I think it would be better to have these in order they will go to them (lifecycle).
How to report issue first with issue template, then followed by PR template. Shouldn't PR template go in contributing guide section?
It should also be stated that PRs are only looked at for approved issues.

Reply:
Done